server/job-workers/job-worker-server.h

```diff
@@ -36,6 +36,8 @@ class JobWorkerServer : vk::not_copyable {
 
   void init();
 
+  void rearm_read_job_fd();
```

Comment:
noexcept?

Reply:
added for all of the JobWorkerServer functions